ethcore/sync/src/chain/mod.rs

```diff
@@ -329,6 +329,10 @@ pub struct PeerInfo {
 	ask_time: Instant,
 	/// Holds a set of transactions recently sent to this peer to avoid spamming.
 	last_sent_transactions: HashSet<H256>,
+	/// Holds a set of private transactions recently sent to this peer to avoid spamming.
+	last_sent_private_transactions: HashSet<H256>,
```

Comment:
Why do we need to separate the two? The collision is not possibly anyway, for simplicity let's have a single set.